includes/class-wc-customer.php

```diff
@@ -118,16 +126,6 @@ public function __construct( $data = 0, $is_session = false ) {
 		}
 	}
 
-	/**
-	 * Prefix for action and filter hooks on data.
-	 *
-	 * @since  3.0.0
-	 * @return string
-	 */
-	protected function get_hook_prefix() {
```

Comment:
Is this intentional? For backward compatibility, we can't remove a protected or public method for generic class like this one..

Reply:
Yes it's intentional. This protected method was redundant because it would return the exact same value as the parent method `\WC_Data::get_hook_prefix()`. There should be no backward compatibility issue because of this parent method.